keygen/src/keygen.rs

```diff
@@ -535,7 +536,7 @@ fn do_main(matches: &ArgMatches<'_>) -> Result<(), Box<dyn error::Error>> {
     let mut wallet_manager = None;
 
     match matches.subcommand() {
```

Comment:
we can just `unwrap()`this, right?  clap will enforce the app definition and bail on unrecognized subcommands or flags before we get here.  probably cleaner than all of the `Some(...)`s